charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -18,4 +18,15 @@ metadata:
 allowVolumeExpansion: true
 provisioner: ebs.csi.aws.com
 volumeBindingMode: WaitForFirstConsumer
+  {{- if semverCompare ">= 1.18" .Values.kubernetesVersion  }}
```

Comment:
You don't need this `if` condition - this file contains the CSI storage classes, and when CSI is used the CSI snapshot components are always deployed as well. Hence, we can also always deploy the `VolumeSnapshotClass`.
The check for the minimum required Kubernetes version (1.18) is already done in the Golang code itself, so no need to take care here again. :)

Reply:
cool Golang code 👍